src/topp/OpenSwathWorkflow.cpp

```diff
@@ -426,15 +425,11 @@ class TOPPOpenSwathWorkflow
     registerStringOption_("enable_ipf", "<name>", "true", "Enable additional scoring of identification assays using IPF (see online documentation)", false, true);
     setValidStrings_("enable_ipf", ListUtils::create<String>("true,false"));
 
-    // one of the following two needs to be set
-    registerOutputFile_("out_features", "<file>", "", "output file", false);
-    setValidFormats_("out_features", ListUtils::create<String>("featureXML"));
-
-    registerOutputFile_("out_tsv", "<file>", "", "TSV output file (mProphet-compatible TSV file)", false);
-    setValidFormats_("out_tsv", ListUtils::create<String>("tsv"));
+    registerOutputFile_("out_features", "<file>", "", "feature output file, either .osw (PyProphet-compatible SQLite file) or .featureXML", false);
+    setValidFormats_("out_features", ListUtils::create<String>("osw,featureXML"));
 
-    registerOutputFile_("out_osw", "<file>", "", "OSW output file (PyProphet-compatible SQLite file)", false);
-    setValidFormats_("out_osw", ListUtils::create<String>("osw"));
+    registerStringOption_("out_features_type", "<type>", "", "input file type -- default: determined from file extension or content\n", false);
+    setValidStrings_("out_features_type", ListUtils::create<String>("osw,featureXML"));
```

Comment:
a bit more modern: if we have more than one entry in the list, we can use the constructor with c++ initalizer-lists.
```suggestion
    setValidStrings_("out_features_type", {"osw","featureXML"});
```